Formula/docker-credential-helper-ecr.rb

```diff
@@ -0,0 +1,25 @@
+class DockerCredentialHelperEcr < Formula
+  desc "Docker Credential Helper for ECR (Amazon EC2 Container Registry)"
+  homepage "https://github.com/awslabs/amazon-ecr-credential-helper"
+  url "https://github.com/awslabs/amazon-ecr-credential-helper/archive/e63ef99.tar.gz"
+  sha256 "35f4cd78df27256f705858c5e33a6fd02882ae2ac50e60348c1ab00e9eb1cc23"
+  head "https://github.com/awslabs/amazon-ecr-credential-helper.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/awslabs/amazon-ecr-credential-helper"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+
+    cd dir do
+      system "make", "build"
+      bin.install "bin/local/docker-credential-ecr-login"
```

Comment:
Need a `prefix.install_metafiles` after this line.

Reply:
What does `prefix.install_metafiles` do, and is it documented anywhere? I don't see any reference to it in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook.html).

When I made this formula, I started from the [docker-credential-helper](https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-credential-helper.rb) formula and modified the relevant bits. Does that formula also need `prefix.install_metafiles`?

Reply:
It's not documented _per se_ because the fact we need to use it in certain formulae is kinda slightly sort of a bug.

Metafiles are things like the [LICENSE, README, CHANGELOG, etc](https://github.com/Homebrew/brew/blob/79e8cdd3ed8c7f5dee573648d55eb3d3e305de19/Library/Homebrew/metafiles.rb), those sorts of documents. Homebrew is pretty good at installing those [automatically](https://github.com/Homebrew/brew/blob/e108cf8cf57835a99a19dc84d26aea3b3cd8917f/Library/Homebrew/build.rb#L149-L151), but sometimes where the contents of the buildpath are transposed to another location that logic can fall on its face.

> Does that formula also need prefix.install_metafiles?

Yes, good spot. https://github.com/Homebrew/homebrew-core/pull/18967

Reply:
Fixed.